### PR TITLE
Package/ozz animation 0.14.1

### PR DESCRIPTION
**Pull request by AlexRamallo**
Specify library name and version:  **ozz-animation/0.14.1**

Add version 0.14.1 and update recipe to be compatible with Conan 2.0

---

- [X] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [X] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [X] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/ozz-animation/all/conanfile.py [hunk 1]
```diff
@@ -1,66 +1,204 @@
...
+            "MinSizeRel": "_rs",
+            "RelWithDebInfo": "_rd",
+        }[str(self.settings.build_type)]
+
+        def _add_libm(c):
+            if self.settings.os in ("Linux", "Android", "FreeBSD"):
+                self.cpp_info.components[c].system_libs = ["m"]
+
+        self.cpp_info.components["base"].libs = [f"ozz_base{postfix}"]
+        self.cpp_info.components["base"].includedirs = ["include"]
```
Comment (valgur) [inline recipes/ozz-animation/all/conanfile.py:170]:
```suggestion
```
Correct, but unnecessary. `["include"]` is the default anyway.

File: recipes/ozz-animation/all/conanfile.py [hunk 2]
```diff
@@ -1,66 +1,204 @@
...
+        "ozz_animation_offline": False,
+        "ozz_animation_tools": False,
     }
-    short_paths = True
 
-    _source_subfolder = "source_subfolder"
-    _build_subfolder = "build_subfolder"
+    def validate(self):
+        def _ensure_enabled(opt, req):
+            if getattr(self.options, opt):
```
Comment (valgur) [inline recipes/ozz-animation/all/conanfile.py:55]:
```suggestion
            if self.options.get_safe(opt):
```
Just a suggestion. `getattr()` is ok, too.

**Review by valgur — APPROVED** [review]

File: recipes/ozz-animation/all/conanfile.py [hunk 3]
```diff
@@ -1,66 +1,204 @@
...
     def package(self):
-        cmake = self._configure_cmake()
+        cmake = CMake(self)
         cmake.install()
 
-        os.remove(os.path.join(self.package_folder, "CHANGES.md"))
-        os.remove(os.path.join(self.package_folder, "LICENSE.md"))
-        os.remove(os.path.join(self.package_folder, "README.md"))
-        self.copy(pattern="LICENSE.md", dst="licenses", src=self._source_subfolder)
+        pkg = Path(self.package_folder)
```
[the remark this answers is not in the document]
Comment (AlexRamallo) [reply recipes/ozz-animation/all/conanfile.py:137]: Interesting, I didn't realize that existed. Is it safe to use? It's not documented, and I noticed that no other recipes use it.

[the remark this answers is not in the document]
Comment (valgur) [reply recipes/ozz-animation/all/conanfile.py:137]: It's somewhat new and undocumented, but should be safe to use:
https://github.com/conan-io/conan/pull/11585#issuecomment-1179091254

**Review by valgur — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Pull request merged**